indexer/cmd/indexer/README.md

```diff
@@ -0,0 +1,26 @@
+# cmd/indexer/main.go
+
+The cli entry point for the indexer.
+
+- **Overview**
+
+`main()` function in [cmd/indexer/main.go](./main.go) is entrypoint to program.   It parses the cli flags to pass into the [indexer](../../indexer.go)
+
+- **Details**
+
+We use [urfave/cli](https://cli.urfave.org/v1/getting-started/) v1 to create a new golang cli.   Note we do not use v2.
```

Comment:
Other parts of the codebase do use v2 fwiw

Reply:
should I update to urfave/cli v2 in the [indexer refresh project](https://www.notion.so/oplabs/Indexer-Refresh-862a5fa978d848038f76a006575e95e6)?